examples/fabric-admin/device_manager/DeviceManager.h

```diff
@@ -149,6 +169,7 @@ class DeviceManager : public PairingDelegate
 
     chip::NodeId mLastUsedNodeId     = 0;
     chip::NodeId mRemoteBridgeNodeId = chip::kUndefinedNodeId;
+    chip::NodeId mLocalBridgeNodeId  = chip::kUndefinedNodeId;
```

Comment:
What is the difference between LocalBridge and RemoteBridge? We should really document this to make it easier for anyone looking at this code to understand

It seems like Remote is when we do some sort of command on fabric-admin from command line and Local is when request comes from CCTRL. Why do we need to distinguish between the two?

Reply:
In E2E, each Fabric-Admin need to pair two bridge to conduct Fabric-Sync, the LocalBridge is the bridge on the same physical device, and RemoteBridge is the bridge on the other ecosystem.